src/reference/01-General-Info/90-Changes/45-sbt-0.13-syntax-migration-guide.md

```diff
@@ -0,0 +1,105 @@
+---
+out: sbt-0.13-Syntax-Migration-Guide.html
```

Comment:
If we follow the convention from http://www.scala-sbt.org/0.13/docs/Migrating-from-sbt-07x.html this page should be `Migrating-from-sbt-012x`?